src/model/DistrictHeating.hpp

```diff
@@ -13,6 +13,8 @@ namespace openstudio {
 
 namespace model {
 
+  class Schedule;
+
   namespace detail {
 
     class DistrictHeating_Impl;
```

Comment:
I'm not opposed if there is an alias. Is there a precedent for alias on Model types anywhere else? Certainly we have them for fields/methods, but I'm not sure about types.

Reply:
I have made that change in #4972. Fully aliased, with a deprecated message in both C++, as well bindings (Ruby and Python)

Yes, we have a precedent, AirTerminalSingleDuctUncontrolled -> AirTerminalSingleDuctConstantVolumeNoReheat. Which we have since 2.7.0 and we should definitely remove now.

Reply:
https://github.com/NREL/OpenStudio/pull/4972/files#diff-ccd34a0619fa7fdf2219b31ac16a85edbbd5c8fae3459e6fb7bee04d67402aa6